### DISCUSSION TITLE
ScriptReference/PrefabUtility.GetPropertyModifications

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#216] [Notes]
# ScriptReference/PrefabUtility.GetPropertyModifications

https://docs.unity3d.com/ScriptReference/PrefabUtility.GetPropertyModifications.html

**Comment by mvi** [comment]
See [`PrefabUtility.IsDefaultOverride`](https://docs.unity3d.com/ScriptReference/PrefabUtility.IsDefaultOverride.html) to filter out modifications made to prefab roots.
